go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/docker/compose/v2
 
 go 1.20
 
+replace github.com/compose-spec/compose-go => ../../../compose-spec/compose-go
```

Comment:
just a reminder here so we don't merge this until a new compose-go release + removing this replace